v1/api/swagger/components/schemas/NewHost.yaml

```
@@ -142,6 +142,10 @@ properties:
     type: object
     additionalProperties:
       type: string
-    description: The map of label name to label value for this host.
+    description: >-
+      The map of service/user specified label name to label value for this host.
+      
+      Setting this field is restricted by role.
```

Should "is" be changed to "may be"? I think either way is ok, since we're not detailing the difference here.
```suggestion
      Setting this field may be restricted by role.
```